styleguide/_themes/derek/scss/components/rsTabs.scss

```diff
@@ -0,0 +1,126 @@
+.rsTabs {
+  display: block;
+  margin-bottom: 10px;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+}
+
+.rsTabs-menu {
+  list-style: none;
+  margin: 0;
+  margin-left: 15px;
+  padding: 0;
+
+  .rsTabs-tabActive {
+    background-color: $rackspace-red;
+    border: 1px solid $rackspace-red;
+    padding-top: 15px;
+    position: relative;
+    z-index: 5;
+
+    &:hover {
+      background-color: $rackspace-red;
+    }
+
+    .rsTabs-tabLink {
+      color: $white;
+    }
```

Comment:
Since .rsTabs-tabActive does something different inside the menu than it does in the content section I nested it inside both. I'll just add another class